js/apps/admin-ui/src/PageHeader.tsx

```diff
@@ -181,20 +197,21 @@ export const Header = () => {
     : toDashboard({ realm });
 
   return (
-    <PageHeader
-      showNavToggle
-      logo={
-        <Link to={logoUrl}>
-          <Brand
-            src={environment.resourceUrl + logo}
-            id="masthead-logo"
-            alt="Logo"
-            className="keycloak__pageheader_brand"
-          />
-        </Link>
-      }
-      logoComponent="div"
-      headerTools={headerTools()}
-    />
+    <Masthead>
+      <MastheadToggle>
+        <PageToggleButton variant="plain" aria-label={t("navigation")}>
+          <BarsIcon />
+        </PageToggleButton>
+      </MastheadToggle>
+      <MastheadBrand href={logoUrl as string}>
```

Comment:
If `logoUrl` is not specified, it is of type `Partial<Path>`, which is not able to be used as a string. Perhaps you can run this through [`useHref()`](https://reactrouter.com/en/main/hooks/use-href) to obtain the actual URL.